### ISSUE TITLE
Hi, Can i use this to block domains like facebook.com? 

### ISSUE THREAD

**Issue by ceojazzy** [mordasov/AsyncProxy#3]
If yes can you please help i am ready to pay for the help. i need to make an app like blocksite (app store).
thanks

**Comment by NikoSquare** [comment]
Nope. This is different type of app. Content filtering apps work differently. I'm not sure at this point, by easiest way may be to try Apple Configurator to create custom device profile.
